src/features/account/components/reviews/ReviewsTabPanel.tsx

```diff
@@ -0,0 +1,12 @@
+import { SimpleGrid } from '@chakra-ui/react';
+import type { FunctionComponent } from 'react';
+
+export const ReviewsTabPanel: FunctionComponent = () => {
+  return (
+    <SimpleGrid
+      columns={[1, null, 2]}
+      spacing="4"
+      data-testid="reviews-tab"
+    ></SimpleGrid>
```

Comment:
```suggestion
    />
```

Reply:
it will have more components in the up coming story